app/file.go

```diff
@@ -1331,7 +1331,7 @@ func (a *App) SearchFilesInTeamForUser(c *request.Context, terms string, userId
 		case errors.As(nErr, &appErr):
 			return nil, appErr
 		default:
-			return nil, model.NewAppError("SearchPostsInTeamForUser", "app.post.search.app_error", nil, nErr.Error(), http.StatusInternalServerError)
+			return nil, model.NewAppError("SearchFilesInTeamForUser", "app.post.search.app_error", nil, nErr.Error(), http.StatusInternalServerError)
```

Comment:
This is not related to this PR, but I changed it since I found it. Not sure if I also should change the error id